crossdock/docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
             - node
             - java
             - python
+            - zipkin-brave-json
+            - zipkin-brave-thrift
 
         environment:
             - WAIT_FOR=test_driver,go,node,java,python
```

Comment:
we should wait for zipkin-brave-json/zipkin just in case (assuming they have a health check endpoint)